test/e2e/tests/collector_pipeline/main_test.go

```diff
@@ -23,7 +23,7 @@ func TestMain(m *testing.M) {
 	// Create KinD Cluster
 	util.Testenv.Setup(
 		envfuncs.CreateKindClusterWithConfig(util.KindClusterName, util.NodeVersion, "../../kind-config.yaml"),
-		envfuncs.CreateNamespace(util.TestNamespace),
+		envfuncs.CreateNamespace(util.EraserNamespace),
```

Comment:
we don't need to create the namespace before any of the tests besides metrics and exclusion list, so we should be able to get rid of this in the rest of the tests